worlds/shivers/__init__.py

```diff
@@ -0,0 +1,194 @@
+from .Items import item_table, ShiversItem
+from .Rules import set_rules
+from BaseClasses import Item, Tutorial, Region, Entrance, Location
+from Fill import fill_restrictive
+from worlds.AutoWorld import WebWorld, World
+from . import Constants, Rules
+from .Options import Shivers_options, get_option_value
+
+client_version = 0
```

Comment:
what is this for?

Reply:
I have no idea, I saw it in a lot of worlds. I am guessing it is not needed for anything. I have deleted it.